web/cobrands/fixmystreet.com/big-green-banner-ab-test.js

```diff
@@ -0,0 +1,52 @@
+(function($){
+    var variation = 0;
+    if ( typeof cxApi !== "undefined" ) {
+        variation = cxApi.chooseVariation();
+    }
+
+    if (variation === 1) {
+        var $original = $('#side .big-green-banner');
+        var $clone = $original.clone();
```

Comment:
This seems like a very complicated way of going about this :) Would it not be far less code to behave as 4afd0026262dd74425d40ac78649db3d8148c883 did – have another `big-green-banner` in the HTML inside `map_box` with class variant1, add class variant0 to the original banner, have this CSS:
```
-html .variant1 { display: none !important; }
-html.variant1 .variant0 { display: none !important; }
-html.variant1 .variant1 { display: block !important; }
```
and set variant1 on `html` if it is that variant. Everything appears/disappears as it should, no more JS.

Reply:
Interesting! I'll give it a try!